docs/source/moc-esi/sso.rst

```diff
@@ -0,0 +1,30 @@
+Single Sign-on Instructions
+===========================
+
+New User Sign-up
+----------------
+New users can access ESI OpenStack `dashboard`_ login page and choose the **OpenID Connect** authentication method. Then you will be redirected to a page to select a login provider. After choosing a login method, follow the steps provided.
+
+Note that during the first login, you will receive a 404 Unauthorized error message. If this occurs, please contact ESI admin to add your user account to a proper project and assign roles to your account.
+
+After your account has been properly configured, you can follow the same steps to log in and successfully authenticate, getting access to the ESI Horizon dashboard.
+
+Use the CLI to Authenticate
+---------------------------
+Once new users have signed up using Horizon WebSSO, they can use the CLI tool to access ESI OpenStack APIs.
+
+Prerequisites
+~~~~~~~~~~~~~
+The **python-openstackclient** is required to authenticate users.
+
+Create an Application Credential
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Log in to ESI `dashboard`_ using your account credentials.
+2. Select the project for which you want to create an application credential from the dropdown menu.
+3. In the sidebar, navigate to **Identity > Application Credentials**.
+4. Click the **Create Application Credential** button.
+5. Provide a name for your credential; all other fields are optional.
+6. After creation, you will be prompted to download an RC file or a clouds.yaml file. Download the file and use it to run OpenStack command-line clients.
```

Comment:
It would probably be a good idea to link to documentation on how exactly to use these files. Maybe [here](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html)? That certainly covers the topic but I don't know if something better exists.

Reply:
Done! I think this link is perfect.